packages/terra-framework-docs/src/terra-dev-site/test/pills/SelectablePills/SelectableAndRemovablePill.test.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useRef, useState } from 'react';
+import Popup from 'terra-popup';
+import { SelectablePills } from 'terra-pills/lib/index';
+import Spacer from 'terra-spacer';
+
+const SelectableAndRemovablePill = () => {
+  const pillsData = [
+    {
+      label: 'asthma',
+      ref: useRef(),
+      id: 'terra-pills-example-disclosure-removable-pill-asthma',
+    },
+    {
+      label: 'bronchitis',
+      ref: useRef(),
+      id: 'terra-pills-example-disclosure-removable-pill-bronchitis',
+    },
+    {
+      label: 'fibro',
+      ref: useRef(),
+      id: 'terra-pills-example-disclosure-removable-pill-fibro',
+    },
+  ];
+  const [pillsState, setPillsState] = useState(pillsData);
+  const [openPillIndex, setOpenPillIndex] = useState(undefined);
```

Comment:
```suggestion
  const [openPillIndex, setOpenPillIndex] = useState(-1);
```
Nit : Can we use -1 instead of `undefined`

Reply:
Undefined works similar.